soh/src/code/z_parameter.c

```diff
@@ -27,6 +27,9 @@
 #define DO_ACTION_TEX_HEIGHT() 16
 #define DO_ACTION_TEX_SIZE() ((DO_ACTION_TEX_WIDTH() * DO_ACTION_TEX_HEIGHT()) / 2)
 
+#define R_ENEMY_HEALTH_BAR_WIDTH 64
```

Comment:
any reason not to throw this in the cosmetics editor as well?

Reply:
As in make it customizable in the cosmetic editor? I feel too small or too large values wouldn't look good on screen. Not sure how/where we would place such a slider in there.

Reply:
In the Hud placement tab was what I was thinking, like we have a slider for heart column width. Not a huge deal tbh just a thought

Reply:
Ok added this and other editor options for anchoring